packages/flutter/test/widgets/media_query_test.dart

```diff
@@ -225,6 +229,33 @@ void main() {
    expect(insideTextScaleFactor, 4.0);
  });
 
+  testWidgets('MediaQuery.platformBrightnessOf', (WidgetTester tester) async {
```

Comment:
(I've not looked into how visible this is):

Any chance we can get a little more test coverage here? E.g. make sure that the appropriate callbacks are called when this value changes from the engine side.

Reply:
I'll look into additional tests.  Please take a look at the next push to see if the test coverage is acceptable.